language/stdlib/modules/Libra.move

```diff
@@ -700,71 +700,159 @@ module Libra {
         Transaction::assert(is_currency<CoinType>(), 1);
     }
 
-    // **************** SPECIFICATIONS ****************
-    // Only a few of the specifications appear at this time. More to come.
+    /// **************** SPECIFICATIONS ****************
+    /// Only a few of the specifications appear at this time. More to come.
 
-    // Verify this module
-    spec module {
+    /// # Module specifications
 
-        // Verification is disabled because of an invariant in association.move that
-        // causes a violated precondition here.  And "invariant module" in association.move
-        // gets an error for some reason.
+    spec module {
 
-        pragma verify = false;
+        /// Verification is disabled because of an invariant in
+        /// RegisteredCurrencies.move that causes a violated precondition here
+        /// and unprovable invariants here.
+        pragma verify = false, aborts_if_is_partial = true;
     }
 
-    // ## Currency registration
-
     spec module {
-        // Address at which currencies should be registered (mirrors currency_addr)
+        /// Address at which currencies should be registered (mirrors currency_addr)
         define spec_currency_addr(): address { 0xA550C18 }
 
-        // Checks whether currency is registered.
-        // Mirrors is_currency<CoinType> in Move, above.
+        /// Address of Treasury Compliance
+        define spec_treasury_compliance_addr(): address { 0xB1E55ED }
+
+        /// Checks whether currency is registered.
+        /// Mirrors is_currency<CoinType> in Move, above.
```

Comment:
If you use backquote on things like is_currency, and for functions add either Self or module name, as in `Self::is_currency<CointType>`, you get a link in the generated documentation to the move function.